views/pages/funding/thinking-of-applying.njk

```diff
@@ -13,23 +14,26 @@
         ) }}
 
         <div class="nudge-up">
-            {% call contentBox(pageAccent) %}
-                <p>Negotiating the world of funding can feel daunting, but we're here to help.</p>
+            <section class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+                {{ breadcrumbTrail(breadcrumbs) }}
+                <div class="s-prose u-constrained-content-wide">
+                    <p>Negotiating the world of funding can feel daunting, but we're here to help.</p>
 
-                <p>Libero voluptatem placeat excepturi error praesentium, cupiditate quis eius hic repellendus, vitae odit eaque magni saepe natus. Temporibus veritatis nisi incidunt quas qui dolorum velit, maiores, quam numquam voluptatibus recusandae nesciunt laudantium omnis molestias dolore totam magni vel distinctio.<p>
-            {% endcall %}
+                    <p>Libero voluptatem placeat excepturi error praesentium, cupiditate quis eius hic repellendus, vitae odit eaque magni saepe natus. Temporibus veritatis nisi incidunt quas qui dolorum velit, maiores, quam numquam voluptatibus recusandae nesciunt laudantium omnis molestias dolore totam magni vel distinctio.<p>
+                </div>
+            </section>
 
             <section class="u-inner">
                 <ul class="flex-grid flex-grid--3up">
-                    {% set themes = cycler('blue', 'pink-dark', 'orange', 'green', 'turquoise', 'pink', 'cyan') %}
+                    {% set themes = cycler('blue', 'pink-dark', 'orange') %}
                     {% for page in [{
-                        "title": "Thinking of applying",
+                        "title": "Is the Big Lottery Fund right for you?",
                         "imageUrl": "placeholders/placeholder-3.jpg"
                     }, {
-                        "title": "What to expect if you apply",
+                        "title": "Find the right funding?",
```

Comment:
Is this meant to be a question?

Reply:
Is _this_ meant to be a question? (good catch!)